server.js

```diff
@@ -2240,14 +2240,16 @@ mapNugetFeed('myget\\/(.*)', 1, function(match) {
   };
 });
 
-// Puppet Forge
-camp.route(/^\/puppetforge\/v\/([^\/]+\/[^\/]+)\.(svg|png|gif|jpg|json)$/,
+// Puppet Forge modules
+camp.route(/^\/puppetforge\/([^\/]+)\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var userRepo = match[1];
-  var format = match[2];
+  var info = match[1]; // either `v`, `dt`, `e` or `f`
+  var user = match[2];
+  var module = match[3];
+  var format = match[4];
   var options = {
     json: true,
-    uri: 'https://forge.puppetlabs.com/api/v1/releases.json?module=' + userRepo
+    uri: 'https://forgeapi.puppetlabs.com/v3/modules/'+user+'-'+module
   };
   var badgeData = getBadgeData('puppetforge', data);
```

Comment:
The badge tag should match the contents of the information. Could you use `puppetforge` for the version, but `downloads` for the downloads, `endorsement`, and `feedback`?